src/pages/library/dataBrowser-utils.js

```diff
@@ -21,22 +21,13 @@ export const uiMessaging = {
 }
 
 export const datasetReleasePolicies = {
-  'TerraCore:NoRestriction': { label: 'NRES', desc: 'No restrictions' },
-  'TerraCore:GeneralResearchUse': { label: 'GRU', desc: 'General research use' },
-  'TerraCore:NPOA': { label: 'NPOA', desc: 'No population origins or ancestry research' },
-  'TerraCore:NMDS': { label: 'NMDS', desc: 'No general methods research' },
-  'TerraCore:GSO': { label: 'GSO', desc: 'Genetic studies only' },
-  'TerraCore:CC': { label: 'CC', desc: 'Clinical care use' },
-  'TerraCore:PUB': { label: 'PUB', desc: 'Publication required' },
-  'TerraCore:COL': { label: 'COL', desc: 'Collaboration required' },
-  'TerraCore:IRB': { label: 'IRB', desc: 'Ethics approval required' },
-  'TerraCore:GS': { label: 'GS', desc: 'Geographical restriction' },
-  'TerraCore:MOR': { label: 'MOR', desc: 'Publication moratorium' },
-  'TerraCore:RT': { label: 'RT', desc: 'Return to database/resource' },
-  'TerraCore:NCU': { label: 'NCU', desc: 'Non commercial use only' },
-  'TerraCore:NPC': { label: 'NPC', desc: 'Not-for-profit use only' },
-  'TerraCore:NPC2': { label: 'NPC2', desc: 'Not-for-profit, non-commercial use only' },
-  releasepolicy_other: { policy: 'SnapshotReleasePolicy_Other', label: 'Other', desc: 'Misc release policies' }
+  'DUO:0000007': { label: 'DS', desc: 'disease specific research' },
```

Comment:
These enums are actually what we support in our schema! So this also fixes a bug for new inputs.

Reply:
Is there something we can add here in a comment to describe how to update this list?

Maybe a comment with a link to the parent attribute (which I think is http://purl.obolibrary.org/obo/DUO_0000042) would be helpful. Or maybe it's purely driven by our schema and it doesn't matter what's in the ontology, as far as the UI is concerned.

Reply:
I think it is purely driven by our schema and doesn't matter what's in the ontology.